From 650d1ab6000b0f177ab6fbb02293c3317cb2dd09 Mon Sep 17 00:00:00 2001
From: Jouni Malinen <j@w1.fi>
Date: Sat, 14 Sep 2024 13:04:14 +0300
Subject: [PATCH] Revert "tests: Copy A3 into NAN SDF Follow-up"

This reverts commit 81322fa43d1d ("tests: Copy A3 into NAN SDF
Follow-up") to allow simplification of the control interface by removing
the external A3 copying.

Signed-off-by: Jouni Malinen <quic_jouni@quicinc.com>
---
 tests/hwsim/test_nan_usd.py | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/tests/hwsim/test_nan_usd.py b/tests/hwsim/test_nan_usd.py
index bad61ffab..356d33419 100644
--- a/tests/hwsim/test_nan_usd.py
+++ b/tests/hwsim/test_nan_usd.py
@@ -238,11 +238,9 @@ def run_nan_usd_followup(dev0, dev1, multi_chan=False):
     vals2 = split_nan_event(ev)
     if vals2['ssi'] != '':
         raise Exception("Unexpected ssi in Follow-up: " + ev)
-    if vals2['a3'] != '51:6f:9a:01:00:00':
-        raise Exception("Unexpected A3 in Follow-up: " + ev)
 
     # Follow-up from subscriber to publisher
-    cmd = "NAN_TRANSMIT handle={} req_instance_id={} address={} a3={} ssi=8899".format(vals['subscribe_id'], vals['publish_id'], addr1, vals2['a3'])
+    cmd = "NAN_TRANSMIT handle={} req_instance_id={} address={} ssi=8899".format(vals['subscribe_id'], vals['publish_id'], addr1)
     if "FAIL" in dev0.request(cmd):
         raise Exception("NAN_TRANSMIT failed")
 
-- 
2.43.0

